tsconfig.json

```diff
@@ -33,6 +32,9 @@
     "noFallthroughCasesInSwitch": true,
     "moduleResolution": "node",
     "baseUrl": "./",
+    "paths": {
```

Comment:
Es gab einen Grund warum wir das getrennt hatten... 🤔Gibt es einen driftigen Grund das zusammen zu legen?

Reply:
Ich habe es kurz in der Commit-Message erwähnt: IntelliJ kommt bei mehreren tsconfig-Dateien durcheinander. Außerdem wird die Infrastruktur etwas weniger komplex und sobald es wirklich verschiedene Versionen der App geben soll kann man es einfach wieder herstellen.

Wir können es aber auch rückgängig machen und ich lege mir lokal einen Patch (Intellij Changelist), den ich bei Commits ignoriere.

Reply:
@kommander Wozu wurde das ganze denn getrennt?

Übrigens bin ich der Auffassung, dass solche Dinge vor einem Merge ausdiskutiert werden sollten @alexanderthieme